functests/dpdk/dpdk.go

```diff
@@ -0,0 +1,117 @@
+package dpdk
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/client"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/pods"
+)
+
+// Entry to find in the pod logfile
+const logEntry = "Accumulated forward statistics for all ports"
+
+var testDpdkNamespace string
+
+func init() {
+	testDpdkNamespace = os.Getenv("DPDK_TEST_NAMESPACE")
+	if testDpdkNamespace == "" {
+		testDpdkNamespace = "dpdk-testing"
+	}
+}
+
+var _ = Describe("dpdk", func() {
+	var _ = Context("Run a sanity test on a worker", func() {
+		It("Should forward and receive packets", func() {
+			var out string
+			p := findPod()
```

Comment:
Wouldn't it make sense for the test to create the pod it wants to test?
@SchSeba WDYT?

Reply:
I will like to leave the test like this because we deploy the application as part of the feautre-deploy (demo folder).

Reply:
I'd like to see these tests as less tied as we can to this repo, so we can eventually make them part of the sriov test suite or some other OpenShift test in the future.

But if not, let's give `findPod` a better name. Like `findDPDKWorkloadPod` or something along that line.

Reply:
sounds good to me @dorzheh can you address this request?